learningequality/ka-lite
# Smoother updates (particularly video updates)

Video updates have a few issues:
- If there's an unexpected error that causes the `videodownload` to cancel itself, a message reporting a "successful" cancellation is shown (should be an error)
- If videos fail to download, nothing is showing in the interface to indicate that, and the final message indicates success (i.e. is green)
- Progress bar was slightly mismatched between current video and total progress

Changes:
- Report both of the above as error messages to the user
- Show, in the progress bar, such errors, when relevant.
- Make progress bar completely consistent.

Testing:
- Download some videos, make sure everything looks good
- Download some videos, and cancel--still should get a "successful cancellation"
- Download some videos, then turn off your internet connection--you should get an error.
